test/integration/no-ui-reporter.js

```diff
@@ -1,6 +1,11 @@
 /*global mocha, console */
 (function() {
 	'use strict';
+
+	if (!mocha || !mocha.reporter || !mocha.reporter('base')) {
```

Comment:
> Note: Non-headless tests (that run on selenium-webdriver) need to wire up to mocha reporter to collect all the results and report them. (See `test-webdriver.js`). But for other tests this can be skipped.